server/controllers/batches.ts

```diff
@@ -25,10 +25,10 @@ import { errorHandler } from "../services/common";
 import { io } from "../index";
 const logger = require("../services/logger");
 const botId = "100000000000000000000001";
-const randomAnimal = "Squirrel Rhino Horse Pig Panda Monkey Lion Orangutan Gorilla Hippo Rabbit Wolf Goat Giraffe Donkey Cow Bear Bison".split(
+const randomAnimal = "Camel Dolphin Elephant Monkey Otter Zebra Horse Giraffe Bear Gorilla Hippo Bison".split(
```

Comment:
Do we need this twice on the server? 

(I know it was already there, but just wondering if there's an opportunity for us to clean it up and remove repeated code?)

Could we keep one of the two and reference the variable in the other location? 

I'd probably choose to keep it only in `utils.ts` instead of `batches.ts` as that seems like a better place to keep common constants.